uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -74,10 +116,18 @@ public void createClientDevice(String clientDeviceId) {
         log.debug("IoT Thing for client device {} is: {}", clientDeviceId, iotThing);
     }
 
-
+    /**
+     * Pass IoT Thing certificate for connection into specified client component.
+     *
+     * @param clientDeviceId string user defined client device id
+     * @param componentId    componentId of MQTT client
+     * @param brokerId       broker id
+     */
     @And("I connect device {string} on {word} to {string}")
     public void connect(String clientDeviceId, String componentId, String brokerId) {
-        //@TODO Implement method
+        final MqttConnectRequest request = getMqttConnectRequest(clientDeviceId, componentId, brokerId);
```

Comment:
It is possible to have more than one connection at the same time ?
For different agents.
For the same agent.

Reply:
yes for both cases